NethServer/Rsync.pm

```diff
@@ -130,8 +130,9 @@ sub prepareRepository {
         my $host = $record->prop('SftpHost');
         my $user = $record->prop('SftpUser');
         my $dir = $record->prop('SftpDirectory');
+        my $port = $record->prop('SftpPort') || '22';
         $rsync_repository = "$user\@$host:$dir";
-        system("ssh $user\@$host 'mkdir -p -- \"$dir\"; touch \"$dir/backup.marker\"'"); 
+        system("ssh -p $port $user\@$host 'mkdir -p -- \"$dir\"; touch \"$dir/backup.marker\"'"); 
```

Comment:
Changes to this file are not needed since the SSH port is configured inside `/etc/ssh/ssh_config`.

Reply:
tested again and indeed it works without this fix